aws/components/lambda/state.ftl

```diff
@@ -128,7 +128,8 @@
                     "default" : "invoke",
                     "invoke" : lambdaInvokePermission(id),
                     "authorise" : lambdaInvokePermission(id),
-                    "authorize" : lambdaInvokePermission(id)
+                    "authorize" : lambdaInvokePermission(id),
+                    "kinesis" : lambdaKinesisPermission(id)
```

Comment:
Can we come up with a role which is less tied to the resource type e.g. "stream"?

Reply:
Also need to make the distinction between the different kiniesis streams if we are adding support for datastreams. Whats the purpose of this role anyway?